CalibTracker/Configuration/python/SiStripGain/SiStripGain_Fake_cff.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-from CalibTracker.SiStripESProducers.fake.SiStripApvGainFakeESSource_cfi import *    
+from CalibTracker.SiStripESProducers.fake.SiStripApvGainFakeESSource_cfi import *
```

Comment:
This file seems to have only a change in whitespace. If so, it should not be in this PR.